server/src/controllers/noises.ts

```
@@ -20,7 +20,7 @@ interface AddSoundRequest extends Request {
 export const addNoise = async (req: AddSoundRequest, res: Response) => {
   try {
     const audioFile = req.file
-
+    console.log(audioFile)
```

ESLint

[error] Unexpected console statement.

(no-console)